.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: Report a very clearly broken issue.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+        - And most importantly, search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
+
+  - type: dropdown
+    attributes:
+      label: Type
+      options:
+        - Crash to desktop
+        - Game behaviour
+        - Performance
+        - Cosmetic
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: What are the steps for reproducing the bug? Any additional details that might be of help?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots or videos
+      description: Share any kind of visuals you can which show the bug or issue taking place.
+      placeholder: You can attach by dragging and dropping the screenshots/videos into this box.
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Version
+      description: The version you encountered this bug on. This is shown at the bottom of the main menu and also at the end of the settings screen.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Logs attachments
+
+        Logs are necessary to further investigate this bug. They are reset every new game session, so you need to ensure the logs are preserved by not starting/restarting osu! after encountering the bug.
+
+        In the case this isn't a "Crash to Desktop" report, to retrieve logs:
+          1. Head on to game settings and click on "Open osu! folder"
+          2. Then open the `logs` folder located there
+
+        Otherwise, logs can *only* be retrieved via file explorer, as running the game will reset them.
+
+        Logs are, by default, located at:
+          - `%AppData%/osu/logs` *(on Windows),*
+          - `~/.local/share/osu/logs` *(on Linux & macOS).*
+          - `Android/data/sh.ppy.osulazer/files/logs` *(on Android)*,
+          - on iOS they can be obtained by connecting your device to your desktop and copying the `logs` directory from the app's own document storage using iTunes. (https://support.apple.com/en-us/HT201301#copy-to-computer)
+
+        If you have relocated your game folder, head into that location and open the `logs` folder.
+
+        Finally, select all log files and drag them into the "Logs" box below.
```

Comment:
```suggestion
        ## Logs

        Attaching log files is required for every reported bug. Instructions how to find the log files are listed below.

        If the game has not yet been closed since you found the bug:
          1. Head on to game settings and click on "Open osu! folder"
          2. Then open the `logs` folder located there

        If the game crashed or has been closed since you found the bug, **do not** restart the game to retrieve the logs as they will reset. You must find them using the file explorer.

        By default logs can be found in:
          - `%AppData%/osu/logs` *(on Windows),*
          - `~/.local/share/osu/logs` *(on Linux & macOS).*
          - `Android/data/sh.ppy.osulazer/files/logs` *(on Android)*,
          - on iOS they can be obtained by connecting your device to your desktop and copying the `logs` directory from the app's own document storage using iTunes. (https://support.apple.com/en-us/HT201301#copy-to-computer)

        However, if you have selected a custom location for the game files via settings, you must find that custom folder and open the `logs` folder there.

        After locating the `logs` folder, select all log files inside and drag them into the "Logs" box below.
```